tools/scripts/generate-deleted-files-list.php

```diff
@@ -72,6 +72,17 @@ function parseLog($logString, &$deletedFiles, $prefix = '') {
   }
 }
 
+/**
+ * Case-sensitive version of `file_exists`.
+ *
+ * Normalizes the inconsistency between Mac/Win (insensitive) and Linux (sensitive).
+ */
+function file_exists_case_sensitive($filename) {
+  // Use glob to list all files in the directory of the given file
+  $files = glob(dirname($filename) . '/*', GLOB_NOSORT);
```

Comment:
Could this just be `glob($filename, GLOB_NOSORT)`?

Reply:
Not on windows at least. It just returns the filename with the casing you gave it. (This script doesn't run on windows as-is - it needs some fudging, but I expect the mac version does the same when filesystem is set to insensitive there.)

But I'm not sure this actually works as intended anyway, because this only normalizes the basename of filename. The rest of filename is still potentially unmatched to the actual casing.